src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -274,6 +319,7 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
                   sortBy={sortBy}
                   onSort={handleSort}
                   actions={actions()}
+                  areActionsDisabled={() => isLoadingManifestExport}
```

Comment:
I think this transition to toast notifications worked out really well and looks slick.  I'm curious whether there would be value to having it coded such that users could start multiple individual exports at the same time though (and not having to disable the actions for every row while a single one downloads).

Reply:
Something else I just noticed is that the export button in the side panel is not disabled during this time that the actions are disabled.

Reply:
I think it might be worth exploring it and creating a separate story if it's not too tough. At first glance it seems a bit tricky.

Aha good catch on the second part. I just pushed a fix so the button is disabled while loading.

Reply:
Yeah, I think it's fine to go out as you have it since I don't think it was specified either way in the actual story, and it does look good.